test/lib/ansible_test/_internal/docker_util.py

```diff
@@ -116,6 +116,60 @@ def get_docker_hostname():  # type: () -> str
     return hostname
 
 
+@cache
+def get_podman_host_ip():  # type: () -> str
+    """Return the IP of the Podman host."""
+    podman_host_ip = socket.gethostbyname(get_podman_hostname())
+
+    display.info('Detected Podman host IP: %s' % podman_host_ip, verbosity=1)
+
+    return podman_host_ip
+
+
+@cache
+def get_podman_default_hostname():  # type: () -> str
+    """Return the default hostname of the Podman service."""
+    try:
+        stdout = raw_command(['podman', 'system', 'connection', 'list', '--format=json'], capture=True)[0]
```

Comment:
Just a note here, `--format` doesn't exist in all versions.  Locally I have 3.3.1 and it exists there.  On a CentOS8 box I have 3.0.2 and it's missing there.